.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
```

Comment:
Test coverage could also be given in case you are only doing a refactoring, and the tests have already been written before. For me it doesn't make any sense to add this line, whatever is written in here can't be taken for granted...